content/source/docs/cloud/integrations/service-now/index.html.md

```diff
@@ -2,192 +2,206 @@
-
-This can be added to your ServiceNow instance from the [ServiceNow Store](https://store.servicenow.com/).
-
-## Terraform Enterprise Setup
-
-Before installing the ServiceNow integration, you need to perform some setup and gather some information in Terraform Enterprise.
-
-1. [Create an organization](../../users-teams-organizations/organizations.html) (or choose an existing organization) where ServiceNow will create new workspaces.
-1. [Create a team](../../users-teams-organizations/teams.html) for that organization called "ServiceNow", and ensure that it has the organization-level ["Manage Workspaces" permission](../../users-teams-organizations/permissions.html#manage-workspaces). You do not need to add any users to this team.
-1. On the "ServiceNow" team's settings page, generate a [team API token](../../users-teams-organizations/api-tokens.html#team-api-tokens). **Save this API token for later.**
-1. If you haven't yet done so, [connect a VCS provider](../../vcs/index.html) for this Terraform organization.
-1. On the organization's VCS provider settings page, find the "OAuth Token ID" for your VCS provider. This is an opaque ID that Terraform Enterprise uses to identify this VCS provider. **Save the OAuth Token ID for later.**
+## Configure Terraform Cloud
+
+Before installing the ServiceNow integration, the Terraform Admin will need to
+perform the following steps to configure and gather information from Terraform
+Cloud.
+
+1. Either [create an
+   organization](../../users-teams-organizations/organizations.html#creating-organizations)
+   or choose an existing organization where ServiceNow will create new
+   workspaces.
+   **Save the Organization Name for later.**
+1. [Create a team](../../users-teams-organizations/teams.html) for that
+   organization called "ServiceNow", and ensure that it has the
+   organization-level ["Manage Workspaces"
+   permission](../../users-teams-organizations/permissions.html#manage-workspaces).
+   You do not need to add any users to this team.
+1. On the "ServiceNow" team's settings page, generate a [team API
+   token](../../users-teams-organizations/api-tokens.html#team-api-tokens).
+   **Save the Team API Token for later.**
+1. If you haven't yet done so, [connect a VCS provider](../../vcs/index.html) to
+   this Terraform organization. This provider will include one or more
+   repositories with Terraform configuration. Any repository that could be
+   connected to a manually-created Terraform Cloud workspace can also be
+   used as a workspace template in the ServiceNow integration.
+1. On your organization's VCS provider settings page (Settings > VCS Providers),
+   find the "OAuth Token ID" for the VCS provider or providers you will be using
+   with the ServiceNow integration. Terraform Cloud uses the OAuth Token ID to
+   identify and authorize the VCS provider.
+   **Save the OAuth Token ID for later.**
```

Comment:
Capitalizing the "token" in OAuth token ID is correct when quoting the UI text, but not correct when just describing the object.